.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Generate release note
+        id: release-note
+        run: |
+          wget -O git-chglog https://github.com/git-chglog/git-chglog/releases/download/0.9.1/git-chglog_linux_amd64
```

Comment:
Just curious, are there no good existing GH actions for `git-chglog` / is this the recommended way to use it as part of a GH action? Always happy to minimize custom scripts we're maintaining!

Reply:
Created a simple local GitHub Action: [`a64d499` (#27)](https://github.com/spinnaker/kustomization-base/pull/27/commits/a64d4992b40c6673a8d4bdbc64d4635ec616d4fb)

Reply:
AFAIW, there's no good GitHub Action for `git-chglog` or good GitHub Action to generate release notes... So one of the ways to simplify the workflow config is to create a local GitHub Action with Dockerfile.

Reply:
Ah okay, sorry for the back-and-forth, but I think if there's no existing GH action out there, it's likely better to just have it as an inline script as in your original commit, since having it as a separate local GH action might imply that it's meant to be used elsewhere? Let me know what you think!

Reply:
> having it as a separate local GH action might imply that it's meant to be used elsewhere

Yeah, people can write the following workflow:

```yaml
jobs:
  release:
    steps:
      - uses: spinnaker/kustomization-base/.github/actions/action-release-note@master
```

This is a completely unconventional approach, but it's true that people can refer to it.

If you have some concerns about it, I'm going to use the original script 👍

Reply:
Oh wow, that's super interesting! Yeah, I don't think we want people to be using Spinnaker's kustomization base as a source for changelog generation GH actions 😂 

If you ever encounter a well-supported alternative in the GH action marketplace, we can consider replacing the script with that at that point!

Thanks again for all your work here ❤️

Reply:
> I don't think we want people to be using Spinnaker's kustomization base as a source for changelog generation GH actions 😂

It's surely true also for me 😀   Then, let's use the original script for now.

> If you ever encounter a well-supported alternative in the GH action marketplace, we can consider replacing the script with that at that point!

BTW, I'm working on developing various, useful GitHub Actions in [actions-ecosystem org](https://github.com/actions-ecosystem) as one of my personal projects. Perhaps I might create one regardless of this project.

Reply:
> BTW, I'm working on developing various, useful GitHub Actions in actions-ecosystem org as one of my personal projects. Perhaps I might create one regardless of this project.

That's awesome, @micnncim, definitely keep us posted on that!